app/uk/gov/hmrc/gform/views/tasklist/task_list.scala.html

```diff
@@ -64,8 +64,11 @@ <h2 class="govuk-heading-s govuk-!-margin-bottom-2">Application incomplete</h2>
         task_section(formTemplate._id, maybeAccessCode, TaskSectionNumber(index), taskSection, statusesLookup)
       }
     }
+    @formTemplate.submitSection.map{ submitSection =>
+        @{task_submit_section(formTemplate._id, maybeAccessCode, TaskSectionNumber(taskList.sections.size +1 ), submitSection, completedSection === taskList.sections.size)}
+    }
   </ol>
-  @if(completedSection === taskList.sections.size) {
+  @if(formTemplate.submitSection.isEmpty && completedSection === taskList.sections.size) {
```

Comment:
I don't think submit button should stay.

There is word  **_always_** in ticket header